src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -63,7 +63,9 @@
 
 (defn create-select [props options]
   [:select (deep-merge {:style select-style} props)
-   (map-indexed (fn [i opt] [:option {:value i} opt]) options)])
```

Comment:
I know this is my doing in the first place, but now this really needs to be combined with create-identity-select